dom0-updates/qubes-dom0-update

```diff
@@ -137,8 +137,18 @@ case ${YUM_ACTION=upgrade} in
     ;;
 esac
 
+echo "Using $UPDATEVM as UpdateVM for Dom0" >&2
+
 if [ "$CHECK_ONLY" == "1" ]; then
     REMOTE_ONLY=1
+    echo "Checking for dom0 updates..." >&2
+else
+    case ${YUM_ACTION=upgrade} in
+    (reinstall|upgrade|upgrade-to|downgrade)
```

Comment:
What about other actions like `install`? Maybe better rely on already existing `$REMOTE_ONLY` which I think already covers exactly the same difference?

Reply:
There is an special message for `list` & `search` further down in the PR. Since only package list is updated during list & search actions (and they are also included in `$REMOTE_ONLY`).

I broke the messages to more explanatory categories (and included `install`). I also added a `(*)` to catch all other actions (with appropriate message).